src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,76 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final int OPERATION = 0;
+    private static final int AMOUNT = 1;
+
+    private String readFromFile(File pathFile) {
+        StringBuilder res = new StringBuilder();
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(pathFile))) {
+            String line = reader.readLine();
+            while (line != null) {
+                res.append(line).append(System.lineSeparator());
+                line = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read from file",e);
+        }
+        return res.toString();
+    }
+
+    private String createReport(String dataForReport) {
+        StringBuilder report = new StringBuilder();
+        int buy = 0;
+        int supply = 0;
+
+        for (String data : dataForReport.split(System.lineSeparator())) {
+            switch (data.split(",")[OPERATION]) {
+                case "buy":
```

Comment:
it is better to create a constant, since you are using this word twice. likewise with similar cases